tests/test_helpers/helpers.ts

```diff
@@ -138,7 +140,13 @@ export function makeTestEnv(mockEnv: Partial<SpreadsheetChildEnv> = {}): Spreads
     raiseError: mockEnv.raiseError || (() => {}),
     askConfirmation: mockEnv.askConfirmation || (() => {}),
   };
+  // TODORAR : type case or create a fake class that does nothing with its drawlayer ?
+  const gridRendererStore = {
+    drawLayer: (renderingContext: GridRenderingContext, layer: LAYERS) => {},
+  } as GridRenderer;
+
   container.inject(NotificationStore, notificationStore);
+  container.inject(GridRenderer, gridRendererStore);
```

Comment:
we could actually inject a mock instance of `RendererStore` to cancel all renderings, not only the grid renderer